tooling/cli/src/dev.rs

```diff
@@ -64,6 +64,9 @@ pub struct Options {
   /// Disable the dev server for static files.
   #[clap(long)]
   pub no_dev_server: bool,
+  /// Specify port for the dev server for static files. Defaults to 1430
+  #[clap(long)]
+  pub dev_server_port: Option<u16>,
```

Comment:
Can we name it `port` instead? Also we should move the default here so the docs is always in sync. `#[clap(long, default_value = 1430]`

Reply:
`port` seems a bit vague, don't you think?

As for the default, I chose not to use clap `default_value` because I thought that `--dev-server-port` have precedence over the env var `TAURI_DEV_SERVER_PORT`, or should it be the other way around?

Reply:
You're right about the default value, my bad :D I didn't see the precedence.

About the argument name... well I don't know, it seems kinda redundant that it's the dev server port.

Reply:
yeah, changed it to `--port`